windows/window_manager.cpp

```diff
@@ -714,27 +756,31 @@ void WindowManager::StartResizing(const flutter::EncodableMap& args) {
   bool bottom = std::get<bool>(args.at(flutter::EncodableValue("bottom")));
   bool left = std::get<bool>(args.at(flutter::EncodableValue("left")));
   bool right = std::get<bool>(args.at(flutter::EncodableValue("right")));
+  bool isDoubleClick =
```

Comment:
I think we should use a new method to implement this functionality instead of increasing it in startResizing

Reply:
Alright

Reply:
Should I make it private or public?